README.md

```diff
@@ -5,86 +5,113 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/canonical/microceph/microceph)](https://goreportcard.com/report/github.com/canonical/microceph/microceph)
 [![Documentation Status](https://readthedocs.com/projects/canonical-microceph/badge/?version=latest)](https://canonical-microceph.readthedocs-hosted.com/en/latest/?badge=latest)
 
-<p align="center">
-<a href="https://snapcraft.io/microceph">MicroCeph</a> is snap-deployed Ceph with built-in clustering.
-</p>
-
 [![Get it from the Snap Store][snap-button]][snap-microceph]
 
-## Table of contents
 
-* [💡 Philosophy](#-philosophy)
-* [🎯 Features](#-features)
-* [⚡️ Quickstart](#%EF%B8%8Fquickstart)
-* [📖 Documentation](#-documentation)
-* [💫 Project & community](#-project--community)
-* [📰 License](#-license)
+MicroCeph is an opinionated command-line tool for deploying and managing Ceph clusters at all scales.
+It reduces deployment and management overhead by simplifying key distribution, service placement,
+and disk administration through a single interface.
+
+Available as a snap, MicroCeph is the easiest tool for admins, developers, and hobbyists to manage clusters.
+
+## Installation
+
+MicroCeph has first-class support as a snap. On snap-ready systems, you can install it on the command line with:
+
+```
+sudo snap install microceph
+```
+
+Disable automatic snap upgrades, so that no unexpected updates change your set-up:
+
+```
+sudo snap refresh --hold microceph
+```
+
+## Basic usage
+
+MicroCeph can deploy a Ceph cluster on a single machine with minimal commands.
+
+First, set up a Ceph cluster on your machine with:
+
+```
+sudo microceph cluster bootstrap
+```
+
+> [!NOTE]  
+> `cluster` is a microceph subcommand for managing clusters.
+
+After setup, add storage to your cluster with:
+
+```
+sudo microceph disk add loop,4G,3
+```
+
+Here, you’ll add three virtual disks (“loop file” disks) of 4 GiB each.
+
+
+Once your cluster is set up and running, you can monitor its status with:
+
+```
+sudo microceph status
+```
+
+Note that there are no spaces between the `disk add` arguments.
```

Comment:
`disk add` also supports space separated bulk disk addition. Hence this may confuse the users. How about `Note that the loop directive is comma separated.` or something like that.

Reply:
Ah, I see. 

However, the intention behind this note was to reassure the user that there are indeed no spaces between the args as one can instinctively add spaces between the comma-separated arguments.
It is already obvious that they are comma-separated, so I don't think we need to mention that. The fact that `disk add` also supports space-separated values is delightful new info but we don't need to mention that either in the tutorial (it's extra info that might distract the user from their learning experience).

Reply:
ack, fair enough.